ledger/caching/src/main/scala/com/daml/caching/DropwizardStatsCounter.scala

```diff
@@ -43,9 +35,9 @@ private[caching] final class DropwizardStatsCounter(
     new CacheStats(
       metrics.hitCount.getCount,
       metrics.missCount.getCount,
-      metrics.loadSuccessCount.getCount,
-      metrics.loadFailureCount.getCount,
-      metrics.totalLoadTime.getCount,
+      0,
+      0,
+      0,
```

Comment:
even though metrics claimed to be unused, but they're provided to the framework we are using, so now this framework will receive these values incorrectly. I am wondering why we use this framework and can we get rid of it?

Reply:
maybe "framework" is a big word, but anyway we do provide wrong values to the class

Reply:
This is the native caffeine metric class, so we cannot selectively drop it without dropping the entire caffeine. 
We are overriding the default collector to ignore certain kinds of metrics. I see it as a legitimate approach to modify the behavior of he library that doesn't suit out needs. 
Besides the snapshot method is not used by the library, but is a client facing method that we only need to override for completeness sake. We could just as well stuff it with zeroes from top to the bottom. I guess CacheStats.empty would be a more honest and less confusing implementation.